components/scripts/lib/build-scan-offline.sh

```diff
@@ -34,9 +34,20 @@ find_build_scan_dump() {
 }
 
 read_build_scan_dumps() {
-  local build_scan_csv
-  echo -n "Extracting build scan data"
-  build_scan_csv="$(invoke_java "$BUILD_SCAN_SUPPORT_TOOL_JAR" extract "0,${build_scan_dumps[0]}"  "1,${build_scan_dumps[1]}")"
+  local build_scan_csv args
+  args=()
+
+  args+=(
+      "extract"
```

Comment:
Why do we have to pass `extract`?

Reply:
This is a consequence of the CLI framework the offline tool uses. It requires there to be a subcommand, and so Sorin decided to name the subcommand "extract".

Reply:
Thanks for the clarification.